app/_src/policies/meshloadbalancingstrategy.md

```diff
@@ -76,7 +76,7 @@ Advanced locality-aware load balancing provides a powerful means of defining how
         - **`AnyExcept`** - traffic will be load balanced to every available zone except those specified in zones list.
         - **`None`** - traffic will not be load balanced to any zone.
       - **`zones`** - list of zone names
-  - **`failoverThreshold.percentage`** - (optional) defines the percentage of live destination dataplane proxies below which load balancing to the next priority starts. Has to be in (0.0 - 100.0] range. If the value is a double number, put it in quotes.
+  - **`failoverThreshold.percentage`** - (optional) defines the percentage of live destination dataplane proxies below which load balancing to the next priority starts. .e.g: If you have this set to 70 and you have 10 dataplane proxies it will start load balancing to the next priority when the number of healthy destinations falls under 7. The value to be in ]0.0 - 100.0[ range (Default 50). If the value is a double number, put it in quotes.
```

Comment:
I don't know this interval notation and it looks weird.

This is what I'm used to from school:

![image](https://github.com/kumahq/kuma-website/assets/2753650/99fa69c7-05fe-4edd-95ef-e798542c7c54)

([source](https://en.wikipedia.org/wiki/Interval_(mathematics)))

Reverting the notation to what was original:

```suggestion
  - **`failoverThreshold.percentage`** - (optional) defines the percentage of live destination dataplane proxies below which load balancing to the next priority starts. .e.g: If you have this set to 70 and you have 10 dataplane proxies it will start load balancing to the next priority when the number of healthy destinations falls under 7. The value to be in (0.0 - 100.0] range (Default 50). If the value is a double number, put it in quotes.
```